packages/host/app/components/matrix/room-message.gts

```diff
@@ -89,6 +138,11 @@ export default class RoomMessage extends Component<Signature> {
     }}
     {{#if this.resources}}
       <AiAssistantMessage
+        {{SendReadReceipt
+          matrixService=this.matrixService
+          message=@message
+          room=@room
+        }}
```

Comment:
Since when a message is streaming the chat autoscrolls to the bottom (which I would find annoying if I were trying to read something earlier in the chat, as it happens constantly), I think this is not actually a problem, but read receipts are usually sent when a message is viewed (`IntersectionObserver`) vs the element inserted. I think we might need to revisit this at some point, just noting.